src/app/(admin)/new_dashboard/components/ChatPopup/index.tsx

```diff
@@ -0,0 +1,102 @@
+"use client";
+
+import React, { useState } from 'react';
+import { Box, IconButton, Paper, Typography } from '@mui/material';
+import { Chat, Close } from '@mui/icons-material';
+import SupportMessageList from '../SupportMessageList';
+import MessageInput from '@/common/components/MessageInput';
+import { Contact } from '../../types';
+
+interface Props {
+  selectedContact: Contact | null;
+}
+
+const ChatPopup = ({ selectedContact }: Props) => {
+  const [isOpen, setIsOpen] = useState(false);
+
+  const toggleChat = () => {
+    setIsOpen(!isOpen);
+  };
+
+  return (
+    <Box
+      sx={{
+        position: 'fixed',
+        bottom: 0,
+        right: 0,
+        width: '100%',
+        display: 'flex',
+        justifyContent: 'flex-end',
+        alignItems: 'flex-end',
+        padding: 2,
+        boxSizing: 'border-box',
+        zIndex: 1000, // Ensures the chat popup is on top of other elements
+      }}
+    >
+      {/* Toggle Button */}
+      <IconButton
+        color="primary"
+        onClick={toggleChat}
+        sx={{
+          backgroundColor: 'primary.main',
+          color: 'white',
+          '&:hover': { backgroundColor: 'primary.dark' },
+        }}
+      >
+        {isOpen ? <Close /> : <Chat />}
+      </IconButton>
```

Comment:
Why not use a speed dial from MUI for showing a menu with options to toggle the chat?